pair_color_cable.py

```diff
@@ -0,0 +1,11 @@
+from test_color_coding import *
+from color_code_manual import print_manual
+
+if __name__ == '__main__':
+    print_manual()

```

Comment:
Rest of this file is about testing. Is it possible to test `print_manual`?

Reply:
yes it is possible, but since it is hard coded manual thought to not to write unit test for this